posthog/queries/app_metrics/serializers.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework import serializers
+
+
+class AppMetricsRequestSerializer(serializers.Serializer):
+    category = serializers.ChoiceField(
+        # Keep in sync with plugin-server/src/worker/ingestion/app-metrics.ts
+        choices=["processEvent", "onEvent", "exportEvents"],
+        help_text="What date to filter the results from",
+        required=True,
+    )
+    date_from = serializers.CharField(
+        default="-30d",
+        help_text="What date to filter the results from. Can either be a date `2021-01-01`, or a relative date, like `-7d` for last seven days, `-1m` for last month, `mStart` for start of the month or `yStart` for the start of the year.",
+    )
+    date_to = serializers.CharField(
+        required=False,
+        help_text="What date to filter the results to. Can either be a date `2021-01-01`, or a relative date, like `-7d` for last seven days, `-1m` for last month, `mStart` for start of the month or `yStart` for the start of the year.",
```

Comment:
could be helpful to know that it's exclusive range

Reply:
This is copy-pasted from trends docs.